src/components/BookMark/BookMark.stories.js

```diff
@@ -0,0 +1,21 @@
+import BookMark from './BookMark.vue';
+
+export default {
+  title: 'Components/BookMark',
+  component: BookMark,
+};
+
+// More on component templates: https://storybook.js.org/docs/vue/writing-stories/introduction#using-args
+const Template = (args) => ({
+  components: { BookMark },
+  setup() {
+    return { args };
+  },
+  template: '<book-mark v-bind="args" />',
+});
+
+export const Default = Template.bind({});
+// More on args: https://storybook.js.org/docs/vue/writing-stories/args
+Default.args = {
+  label: 'Convocatoria',
+};
```

Comment:
same here

Reply:
I see it on my code editor, maybe github is trim() it
![image](https://user-images.githubusercontent.com/66505715/173159456-e24df01e-45ff-4348-a247-b77abd374d4d.png)